src/main/scala/xiangshan/Bundle.scala

```diff
@@ -162,6 +162,7 @@ class CtrlFlow(implicit p: Parameters) extends XSBundle {
   val ftqPtr = new FtqPtr
   val ftqOffset = UInt(log2Up(PredictWidth).W)
   val isLastInFtqEntry = Bool()
+  val seqNum = UInt(64.W)
```

Comment:
Is there a switch for `seqNum`? Will this modification impact the area?

Reply:
seqNum only used by dpic function, it should be dead code eliminated